src/components/Sections/SectionNovelty/index.scss

```diff
@@ -0,0 +1,42 @@
+.novelty {
+    margin-top: 90px;
+    margin-bottom: 90px;
+
+    &__box{
+        display: flex;
+        gap: 30px; 
+    }
+
+    &__picture{
+        height: auto;
+        width: 100%;
+    }
+
+    &__describe {
+        margin-top: auto;
```

Comment:
please, could you explain what margin-top: auto; and margin-bottom: auto; will do?